docs/api.md

```diff
@@ -307,6 +322,49 @@ If Google Chrome (rather than Chromium) is preferred, a [Chrome Canary](https://
 >
 > See [`this article`](https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/) for a description
   of the differences between Chromium and Chrome. [`This article`](https://chromium.googlesource.com/chromium/src/+/lkcr/docs/chromium_browser_vs_google_chrome.md) describes some differences for Linux users.
+  
+### class: Downloader
+
+A Downloader can download and manage different versions of Chromium.
+
+The Downloader operates on revision strings that specify a precise version of Chromium, e.g. `"533271"`. Revision strings can be obtained from [omahaproxy.appspot.com](http://omahaproxy.appspot.com/).
+
+> **NOTE** Downloader class is not designed to work concurrently with other instances of Downloader class.
+
+#### downloader.canDownload(revision)
+- `revision` <[string]> a revision to check availability.
+- returns: <[Promise]<[boolean]>>  returns `true` if the revision could be downloaded from the host.
+
+The method initiates a HEAD request to check if the revision is available.
+
+#### downloader.download(revision[, progressCallback])
+- `revision` <[string]> a revision to download.
+- `progressCallback` <[function]([number], [number])> A function that will be called with two arguments:
+  - `downloadedBytes` <[number]> how many bytes have been downloaded
+  - `totalBytes` <[number]> how large is the total download.
+- returns: <[Promise]> Resolves when the revision is downloaded and extracted
+
+The method initiates a GET request to download the revision from the host.
+
+#### downloader.platform()
+- returns: <[string]> Downloader target platform. Returns one of `mac`, `linux`, `win32` or `win64`.
+
+#### downloader.remove(revision)
+- `revision` <[string]> a revision to remove. The method will throw if the revision has not been downloaded.
+- returns: <[Promise]> Resolves when the revision has been removed.
+
+#### downloader.revisionInfo(revision)
+- `revision` <[string]> a revision to get info for.
+- returns: <[Object]>
+  - `revision` <[string]> the revision the info was created from
+  - `folderPath` <[string]> path to the extracted revision folder
+  - `executablePath` <[string]> path to the revision executable
+  - `url` <[string]> URL this revision can be downloaded from
+  - `downloaded` <[boolean]> wether the revision has been downloaded or not
```

Comment:
if i download and then remove a rev, what will revInfo.downloaded tell me?  does this property really mean "currently on disk" ?  maybe `available`?

also if i do any of these actions in separate downloader instances (assuming same host/path/platform), does it matter?

Reply:
> if i download and then remove a rev, what will revInfo.downloaded tell me? does this property really mean "currently on disk" ? 

Yes, that's 'currently on disk'.

> also if i do any of these actions in separate downloader instances (assuming same host/path/platform), does it matter?

There are no guarantees in this case. If two downloaders try to download the same revision simultaneously, they will fight with each other and fail in some weird way.

There's a note about this in the description of downloader class.